addons/im_livechat/demo/im_livechat_channel/im_livechat_session_1.xml

```diff
@@ -11,7 +11,8 @@
         <record id="im_livechat.livechat_channel_session_1_member_admin" model="discuss.channel.member">
             <field name="partner_id" ref="base.partner_admin"/>
             <field name="channel_id" ref="im_livechat.livechat_channel_session_1"/>
-            <field name="is_pinned">False</field>
+            <field eval="DateTime.today()" name="unpin_dt"/>
+            <field eval="DateTime.today() + relativedelta(months=-1, days=-0, minutes=1)" name="last_interest_dt"/>
```

Comment:
which such a specific delta? can't we just use one of the values?

Reply:
yes we can, just in case we want to sort those threads in a way. but can be just the same value for now